eng/pipelines/libraries/helix-queues-setup.yml

```
@@ -115,7 +115,11 @@ jobs:
 
     # iOS devices
     - ${{ if in(parameters.platform, 'iOS_arm64') }}:
-      - OSX.1015.Amd64.Iphone.Open
+      # split traffic for PR's and rolling builds
```

nit: this is a bit confusing since you can run the `runtime-extra-platforms` on PRs. I'd use something like this
```suggestion
      # split traffic for runtime-extra-platforms (which mostly runs on rolling builds)
```